src/components/appointments/SubmitRow.js

```diff
@@ -20,10 +21,11 @@ const SubmitRow = ({canSubmit, nextText, previousText = '', navigateBackTo = ''}
       isLastStep={false}
       isCheckingLogic={false}
       loginRequired={false}
+      hideAbortButton={true}
```

Comment:
```suggestion
      hideAbortButton
```
same effect, but is more idiomatic React